packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -31,6 +31,13 @@ import { SCHEMA_MODULE_CONFIG } from './config';
 import { SchemaHelper } from './schema-helper';
 import { HiveError } from '../../../shared/errors';
 import { ArtifactStorageWriter } from './artifact-storage-writer';
+import { GraphQLError } from 'graphql';
+
+import Redlock from 'redlock';
+
+const redlock = new Redlock([null as any], {
```

Comment:
for the pseudo code like demonstration i wanted to avoid all the dependency injection overhead